db/src/db.rs

```diff
@@ -731,14 +732,19 @@ impl DB {
             .chain_err(|| "Could not update datoms: failed to retract datoms already present")?;
 
         // Insert datoms that were added and not already present.
-        let s = r#"
+        let s = format!(r#"
```

Comment:
Yeah, this is early days.  We're still deep in the "make it correct" phase, but I was hoping to get some idea of how much this saves.  We'll be better able to benchmark as the transactor fleshes out.